notebooks/toc.yaml

```diff
@@ -1,364 +1,31 @@
-    - title: Python and Jupyter Notebooks
-      id: introduction-to-python-and-jupyter-notebooks
-      url: /ch-prerequisites/python-and-jupyter-notebooks
-
-
-- title: Quantum States and Qubits
-  url: /ch-states
-  sections:
-    - title: Introduction
-      id: introduction
-      url: /ch-states/introduction
-    - title: The Atoms of Computation
-      id: the-atoms-of-computation
-      url: /ch-states/atoms-computation
-    - title: Representing Qubit States
-      id: representing-qubit-states
-      url: /ch-states/representing-qubit-states
-    - title: Single Qubit Gates
-      id: single-qubit-gates
-      url: /ch-states/single-qubit-gates
-    - title: The Case for Quantum
-      id: the-case-for-quantum-computers
-      url: /ch-states/case-for-quantum
-
-
-- title: Multiple Qubits and Entanglement
-  url: /ch-gates
-  sections:
-    - title: Introduction
-      id: introduction
-      url: /ch-gates/introduction
-    - title: Multiple Qubits and Entangled States
-      id: multiple-qubits-and-entangled-states
-      url: /ch-gates/multiple-qubits-entangled-states
-    - title: Phase Kickback
-      id: phase-kickback
-      url: /ch-gates/phase-kickback
-    - title: More Circuit Identities
-      id: basic-circuit-identities
-      url: /ch-gates/more-circuit-identities
-    - title: Proving Universality
-      id: proving-universality
-      url: /ch-gates/proving-universality
-    - title: Classical Computation on a Quantum Computer
-      id: classical-computation-on-a-quantum-computer
-      url: /ch-gates/oracles
-
-
-- title: Quantum Protocols and Quantum Algorithms
-  url: /ch-algorithms
-  sections:
-    - title: Defining Quantum Circuits
-      id: quantum-circuits
-      url: /ch-algorithms/defining-quantum-circuits
-    - title: Deutsch-Jozsa Algorithm
-      id: deutsch-jozsa-algorithm
-      url: /ch-algorithms/deutsch-jozsa
-    - title: Bernstein-Vazirani Algorithm
-      id: bernstein-vazirani-algorithm
-      url: /ch-algorithms/bernstein-vazirani
-    - title: Simon's Algorithm
-      id: simons-algorithm
-      url: /ch-algorithms/simon
-    - title: Quantum Fourier Transform
-      id: quantum-fourier-transform
-      url: /ch-algorithms/quantum-fourier-transform
-    - title: Quantum Phase Estimation
-      id: quantum-phase-estimation
-      url: /ch-algorithms/quantum-phase-estimation
-    - title: Shor's Algorithm
-      id: shors-algorithm
-      url: /ch-algorithms/shor
-    - title: Grover's Algorithm
-      id: grovers-algorithm
-      url: /ch-algorithms/grover
-    - title: Quantum Counting
-      id: quantum-counting
-      url: /ch-algorithms/quantum-counting
-    - title: Quantum Teleportation
-      id: quantum-teleportation
-      url: /ch-algorithms/teleportation
-    - title: Superdense Coding
-      id: superdense-coding
-      url: /ch-algorithms/superdense-coding
-    - title: Quantum Key Distribution
-      id: quantum-key-distribution
-      url: /ch-algorithms/quantum-key-distribution
```

Comment:
just wanted to call this out. please be sure to properly restore the `toc.yaml` before merging this PR.

Reply:
Whoops sorry must have accidentally committed that